test/testrunner.py

```diff
@@ -309,7 +309,7 @@ def main():
             if specific_test_type == "ec2":
                 pytest_cmd += ["--reruns=1", "--reruns-delay=10"]
             if is_pr_context():
-                pytest_cmd.append("--timeout=4860")
+                pytest_cmd.append("--timeout=10800")
```

Comment:
Can you change this back? This PR build time limit is by design. If the smdebug test really needs 3 hours to run, we should figure out a way to decrease this test time for the functional test, as that's not an appropriate runtime for PR test

Reply:
Reverted